src/ArcherContainer.js

```diff
@@ -217,13 +217,13 @@ export class ArcherContainer extends React.Component<Props, State> {
   };
 
   _registerChild = (id: string, ref: HTMLElement): void => {
-    if (!this.state.refs[id]) {
-      this.state.observer.observe(ref);
+    // if (!this.state.refs[id]) {
```

Comment:
I don't understand this, could you provide some explanation?
It's not related to your issue, is it?

Reply:
Interesting. I thought I wouldn't be pushing this. Basically, I was trying to use react-archer inside of react-window VariableSizedGrid to render a swim lane. 

react-window re-renders the grid when I update any value from outside. I found that the arrows would go away if I updated the state and come back if I updated the state again. 

I investigated further to see that refs were empty in the first time I update the state. I had a reason to believe that since registerChild and unRegisterChild happen quickly, it's registering the first time properly. Commenting out the if clause works properly. 

I wish I could have given you a better explanation, but that's the best I could do.